nnsvs/model.py

```diff
@@ -74,14 +70,52 @@ def __init__(self, in_dim, hidden_dim, out_dim, num_layers=1, bidirectional=True
         super(LSTMRNN, self).__init__()
         self.hidden_dim = hidden_dim
         self.num_layers = num_layers
+        self.relu = nn.ReLU()
         self.num_direction =  2 if bidirectional else 1
         self.lstm = nn.LSTM(in_dim, hidden_dim, num_layers,
             bidirectional=bidirectional, batch_first=True, dropout=dropout)
         self.hidden2out = nn.Linear(self.num_direction*self.hidden_dim, out_dim)
-
```

Comment:
Same as above

Reply:
I added new line there in the latest commit.